### DISCUSSION TITLE
pyscript and pyarmor?

### DISCUSSION THREAD

**Discussion by laboratoriodosfundos** [pyscript/pyscript#1652] [General]
Can anyone tell me if it's possible to use obfuscated .py files with pyarmor?

**Comment by fpliger** [comment]
It's more a question of whether Pyodide supports it or not. In that case, @hoodmane might know more.... but I'd suggest just trying it out and reporting the results 😉

**Reply by hoodmane** [reply]
According to the docs it requires a binary extension, but it's not clear how the binary extension is made.
